data/wv/legislature/Glenn-Jeffries-de2f5e4b-192a-463d-91b0-73f212e8f953.yml

```diff
@@ -6,17 +6,19 @@ gender: Male
 email: glenn.jeffries@wvsenate.gov
 image: http://www.wvlegislature.gov/images/members/2021/senate/jeffries_glenn.jpg
 party:
-- name: Democratic
```

Comment:
@bfossen-ce this is a Senator who's term ends on 2024-12-01. On December 1st of this year they announced they were switching parties from Dem to Rep. I just want to confirm that this edit matches that context. It looks to me like it will list him as a Democrat (line 11) and a Republican (Line 2) as well as have his end date as later this month (line 9).